packages/locale/src/TranslatedHtml.spec.tsx

```diff
@@ -4,11 +4,11 @@ import React from 'react';
 
 import { getStoreConfig } from '@bigcommerce/checkout/test-utils';
 
-import LocaleContext, { LocaleContextType } from './LocaleContext';
-import TranslatedHtml from './TranslatedHtml';
+import * as LocaleContext from './LocaleContext';
```

Comment:
```suggestion
import LocaleContext, { LocaleContextType } from './LocaleContext';
```
Can we avoid `import *` as we prefer importing things explicitly?